packages/plugin-image/src/ImageHtmlDomRenderer.ts

```diff
@@ -1,11 +1,13 @@
 import { AbstractRenderer } from '../../plugin-renderer/src/AbstractRenderer';
 import { ImageNode } from './ImageNode';
 import { HtmlDomRenderingEngine } from '../../plugin-html/src/HtmlDomRenderingEngine';
+import { Youtube } from '../../plugin-youtube/src/Youtube';
 
 export class ImageHtmlDomRenderer extends AbstractRenderer<Node[]> {
     static id = HtmlDomRenderingEngine.id;
     engine: HtmlDomRenderingEngine;
-    predicate = ImageNode;
+    predicate = (node): boolean =>
+        !![ImageNode, Youtube].find(NodeClass => node instanceof NodeClass);
```

Comment:
Ah ah right, I guess `Youtube` is the *plugin* rather than the node. The `instanceof` construction works because it does not typecheck, while the `is` one would have revealed the error.